hack/generate-standard-manifests.sh

```diff
@@ -26,9 +26,9 @@ Generate standard YAML manifests for Antrea using Helm and writes them to output
         --out <DIR>                   Output directory for generated manifests
         --help, -h                    Print this message and exit
 
-In 'release' mode, environment variables IMG_NAME and IMG_TAG must be set.
+In 'release' mode, environment variables AGENT_IMAGE, CONTROLLER_IMAGE, and IMG_TAG must be set.
 
-In 'dev' mode, environment variable IMG_NAME can be set to use a custom image.
+In 'dev' mode, environment variables AGENT_IMAGE & CONTROLLER_IMAGE can be set to use a custom image.
```

Comment:
Is it possible to set AGENT_IMAGE & CONTROLLER_IMAGE with the current code change? Do you mean these values can be unset in dev mode?

Reply:
i didn't get your point.